### DISCUSSION TITLE
unstable_httpBatchStreamLink has problems returning null. Is it a bug?

### DISCUSSION THREAD

**Discussion by capaj** [trpc/trpc#5944] [Q&A]
I have a query on BE like this

```ts
me: trpcContext.procedure.query(async ({ ctx }) => {
  return null
})
```

```ts
  const [trpc] = useState(() =>
    trpcClient.createClient({
      links: [
        ...trpcLinks,
        httpBatchLink({
          url: VITE_API_URL,
          transformer: SuperJSON

        })
      ]
    })
  )
```

everything works ok

when I use unstable_httpBatchStreamLink, I get these errors. When I return false or some object it works ok

![image](https://github.com/user-attachments/assets/7a84e9cc-5ec9-43b4-bf6f-9295b77df1cd)

If this is a bug I may be open to file it as a bug. Can someone confirm this is indeed a bug?

**Comment by KATT** [comment]
If you can reproduce it, file a bug report

**Reply by capaj** [reply]
https://github.com/trpc/trpc/issues/5945
